services/brig/src/Brig/Team/API.hs

```diff
@@ -239,20 +265,33 @@ createInvitation' tid mUid inviteeRole mbInviterUid fromEmail body = do
   lift $ do
     iid <- maybe (liftIO DB.mkInvitationId) (pure . Id . toUUID) mUid
     now <- liftIO =<< view currentTime
-    timeout <- setTeamInvitationTimeout <$> view settings
-    (newInv, code) <-
+    timeout <-
+      if migratePersonalUser
+        then pure $ Timeout $ nominalDay * 2 -- todo(Leif): read from settings?
```

Comment:
yes! or just take `setTeamInvitationTimeout` no matter whats in `migratePersonalUser`.